### PR TITLE
add c comp and irm

### PR DESCRIPTION
**Pull request by coburncoburn**

### CODE REVIEW

File: networks/mainnet.json [hunk 1]
```diff
@@ -190,30 +187,29 @@
             "description": "Standard Comptroller Impl"
         }
     },
-
     "Comp": {
         "contract": "Comp",
         "symbol": "COMP",
         "name": "Compound Governance Token",
         "decimals": 18,
         "address": "0xc00e94Cb662C3520282E6f5717214004A7f26888"
```
Comment (torreyatcitty) [inline networks/mainnet.json:195]: Can we update the name from `"Compound Governance Token"` to `"Compound"`

**Review by torreyatcitty — CHANGES_REQUESTED** [review]
change name of comp token

**Review by jflatow — APPROVED** [review]

File: networks/mainnet.json (hunk 1 above)
Comment (coburncoburn) [reply networks/mainnet.json:195]: This is already on master this way. Are the underlying Tokens used in the app?

Comment (torreyatcitty) [reply networks/mainnet.json:195]: Yeah, we used in the dapp to show the token name

Comment (coburncoburn) [reply networks/mainnet.json:195]: I agree though it is indeed wrong.

**Pull request merged**